CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## Version 1.16.0
+* Release date: August 24, 2022
+* Release status: GA
+
+### What's new in 1.16.0
+* Added support for creating Azure Functions with SQL Bindings from Views in the Object Explorer
+* Fixed issues with Integrated authentication in Unix environments https://github.com/microsoft/vscode-mssql/issues/17333
```

Comment:
Do I need to change the capitalization here too? @Charles-Gagnon

Reply:
For Authentication? I don't think so - the official docs don't. 

https://docs.microsoft.com/en-us/aspnet/web-api/overview/security/integrated-windows-authentication